### DISCUSSION TITLE
The security token included in the request is invalid.

### DISCUSSION THREAD

**Discussion by arustguy** [brefphp/laravel-bridge#82] [General]
I'm getting the below issue with the latest laravel and laravel-bridge latest.

```
Error executing "SendMessage" on "https://sqs.ap-southeast-1.amazonaws.com/{id}/testing-queue"; AWS HTTP error: Client error: `POST https://sqs.ap-southeast-1.amazonaws.com/{id}/testing-queue` resulted in a `403 Forbidden` response:
<?xml version="1.0"?><ErrorResponse xmlns="http://queue.amazonaws.com/doc/2012-11-05/"><Error><Type>Sender</Type><Code>I (truncated...)
InvalidClientTokenId (client): The security token included in the request is invalid. - <?xml version="1.0"?><ErrorResponse xmlns="http://queue.amazonaws.com/doc/2012-11-05/"><Error><Type>Sender</Type><Code>InvalidClientTokenId</Code><Message>The security token included in the request is invalid.</Message><Detail/></Error><RequestId>{request_id}</RequestId></ErrorResponse>
```

Anyone having issue like me?

My IAM has full administrator access and my testing repo looks exactly like the https://github.com/brefphp/examples/tree/master/Laravel/queues except mine is using the latest laravel and bref latest.

Thanks

**Comment by mnapoli** [comment]
I see two possibilities because we need more details to help you (e.g. serverless.yml).

1. Did you add the `AWS_SESSION_TOKEN` variable like described in the readme?
2. Did you authorize Lambda to send messages to the SQS queue? It doesn't matter what permissions your user has on your laptop, you need to authorize Lambda to hit SQS in serverless.yml

**Reply by arustguy** [reply]
Hi @mnapoli I'm sorry. Please close this because this is only happening in my linux PC and can't reproduce it in my mac.
